draft-ietf-scitt-architecture.md

```diff
@@ -216,15 +216,15 @@ An Issuer may be the owner or author of Artifacts, or an independent third party
 
 Non-equivocation:
 
-: a state where it is impossible for a Transparency Service to provide different views of its append-only log to Verifiers about the same Artifact.
+: a state where it is impossible for a Transparency Service to provide different views of its Append-only Log to Verifiers about the same Artifact.
 Over time, an Issuer may register new Signed Statements about an Artifact in a Transparency Service with new information. However, the consistency of a collection of Signed Statements about the Artifact can be checked by all Verifiers.
 
 Receipt:
 
-: a Receipt is a cryptographic proof that a Signed Statement is recorded in the Append-only Log.
-Receipts are based on Signed Inclusion Proofs as described in COSE Signed Merkle Tree Proofs {{-COMETRE}}.
-Receipts can be built on different verifiable data structures, not just binary merkle trees.
-Receipts consist of Transparency Service-specific inclusion proofs, a signature by the Transparency Service of the state of the Append-only Log, and additional metadata (contained in the signature's protected headers) to assist in auditing.
+: a cryptographic proof that a Signed Statement is included in the Append-only Log.
+Receipts are based on Signed Inclusion Proofs as described in COSE Signed Merkle Tree Proofs {{-COMETRE}};
+they can be built on different verifiable data structures, not just binary merkle trees.
```

Comment:
Isn't this the only one we have defined? Why not put them in a table and make it small changes to add if there is presumably more than one? It seems we have COMETRE and the CCF profile for COMETER, so I am not sure why we would want to emphasize this when no public draft or IETF spec drafts an alternative one.

Am I missing something?

Reply:
While COMETRE is currently the only one defined, the spec allows for more. Not sure the spec needs to list all the current and potential future implementations. 
To get this merged, I'm deferring this to a potential new PR, if felt needed.